docs/src/main/sphinx/functions/table.rst

```diff
@@ -43,6 +43,31 @@ Built-in table functions
     The argument ``input`` is a table or a query.
     The argument ``columns`` is a descriptor without types.
 
+.. function:: sequence(start => bigint, stop => bigint, step => bigint) -> table(sequential_number bigint)
```

Comment:
@mosabua @colebow can you please add this to https://trino.io/docs/current/functions/list.html#s?

same for exclude columns above